src/utils/equipment-types.js

```diff
@@ -194,19 +218,23 @@ export const filterEquipmentDefinition = {
             ...Countries1,
             ...Countries2,
             ...NominalVoltage,
+            freeProperties1: FreeProps.freeProperties,
+            freeProperties2: FreeProps.freeProperties,
```

Comment:
```suggestion
            ...FreeProps1,
            ...FreeProps2,
```

Reply:
This will fix wrong labels in hvdc filders (we have 2 times the same text, not "... props 1" and "... props 2")